src/WebJobs.Script/Description/NodeFunctionInvoker.cs

```diff
@@ -297,6 +297,15 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
             inputDictionary["method"] = request.Method.ToString().ToUpperInvariant();
             inputDictionary["query"] = request.GetQueryNameValuePairs().ToDictionary(kv => kv.Key, kv => kv.Value, StringComparer.OrdinalIgnoreCase);
 
+            Dictionary<string, string> headers = new Dictionary<string, string>();
```

Comment:
Ah, I use var sometimes, but I think this is a personal preference thing. We have pretty strict stylecop in place in this repo and it likes my code, so I'll leave as is.